.github/workflows/github-actions.yml

```diff
@@ -8,8 +8,8 @@ on:
   push:
     branches:
       - "pull-request/[0-9]+"
-      - "branch-*"
```

Comment:
This repo doesn't current use any `branch-*` branches. I suspect this was maybe copied from somewhere else in RAPIDS. Removing it to simplify configuration a bit.